### PR TITLE
Add "Leslie the rhino" to /assets page

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Add Leslie the rhino to the assets page to be downloaded
- Add Tomo Saito artist info

### CODE REVIEW

**Comment by corwintines** [comment]
hmm, not sure whats going on with all these extra commits. Will look into this tomorrow

**Comment by corwintines** [comment]
File changes are correct though

File: src/pages/assets.tsx [hunk 1]
```diff
@@ -275,6 +282,24 @@ const AssetsPage = ({ data }: PageProps<Queries.AssetsPageQuery, Context>) => {
...
+            artistUrl="https://tomosaito.com/"
+            image={data.leslieTheRhino}
+          />
+          <AssetDownload
+            title={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            alt={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            artistName="Tomo Saito"
+            artistUrl="https://tomosaito.com/"
+            image={data.leslieTheRhino}
+            shouldHide={true}
```
Comment (minimalsm) [inline src/pages/assets.tsx:299]: Is this just for left aligning the image container? Seems weird but looks like we already implemented it...

**Review by minimalsm — APPROVED** [review]

File: src/pages/assets.tsx (hunk 1 above)
Comment (corwintines) [reply src/pages/assets.tsx:299]: Without it the image is full width and huge. I just used another example of this from where the wallet robot is added.

**Pull request merged**